.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.vscode/settings.json
```

Comment:
I think you can ignore the whole folder here.

Reply:
Even though I definitely have this committed in other phosphor repos :sweat_smile: